src/blocks/common.blocks/content/_view/content_view_block.bemtree

```diff
@@ -1 +1,3 @@
-block('content').mod('view', 'block').content()({ block: 'block' });
+block('content').mod('view', 'block').content()(function () {
```

Comment:
Because of  `enb-bemxjst` update. There is error `Only literal or function is allowed in template's body` in code like this.
That's why I rewrite some templates.
/cc @andrewblond